scene/register_scene_types.cpp

```diff
@@ -1099,6 +1099,8 @@ void register_scene_types() {
 	ClassDB::add_compatibility_class("StreamTexture2DArray", "CompressedTexture2DArray");
 	ClassDB::add_compatibility_class("StreamTexture3D", "CompressedTexture3D");
 	ClassDB::add_compatibility_class("StreamTextureLayered", "CompressedTextureLayered");
+	ClassDB::add_compatibility_class("Position2D", "Marker2D");
+	ClassDB::add_compatibility_class("Position3D", "Marker3D");
```

Comment:
These should be moved in the previous block. This one is specifically for things renamed between two alpha builds, not since 3.x.

Reply:
Fixed!